.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
It is good to add a `.gitignore` file to any serious project.
Here I am ignoring JetBrains IDE configs -- those are personal and don't need to be shared and clutter up the project.
This also makes it easy to do `git add -A` to quickly stage changes without having to hunt and pick.

Reply:
Done.